examples/useCases/insituGlobalArrays/CMakeLists.txt

```
@@ -3,16 +3,42 @@
 # accompanying file Copyright.txt for details.
 #------------------------------------------------------------------------------#
 
-add_executable(insituGlobalArraysWriter insituGlobalArraysWriter.cpp)
-target_link_libraries(insituGlobalArraysWriter adios2::cxx11)
+cmake_minimum_required(VERSION 3.12)
+project(ADIOS2UseCasesInSituGlobalArraysExample)
 
-add_executable(insituGlobalArraysReaderNxN insituGlobalArraysReaderNxN.cpp)
-target_link_libraries(insituGlobalArraysReaderNxN adios2::cxx11)
+if(NOT TARGET adios2::cxx11)
+  set(_components C CXX)
+
+  find_package(MPI COMPONENTS ${_components})
```

```suggestion
  find_package(MPI COMPONENTS C)
```
I think that we only need the MPI C components.

---

In some tests we need the fortran also, so use the ``_components`` arguments to pass the same component parts to both MPI and ADIOS2 for simplicity.